### ISSUE TITLE
Why not mean-std normalization for the input image?

### ISSUE THREAD

**Issue by glhfgg1024** [CSAILVision/semantic-segmentation-pytorch#43]
Hi thank you very much for this great toolkit! I have a question, I found in semantic segmentation or ImageNet based image classification, the input image was commonly subtracted by the global mean for each channel (RGB) in training set, as shown [here](https://github.com/CSAILVision/semantic-segmentation-pytorch/blob/master/dataset.py#L34).
Why didn't the image be divided by the standard deviation for each channel?

**Comment by Tete-Xiao** [comment]
This is because that we didn't do that when training the ImageNet model.

Dividing the input by its std has become much less useful when we have so many novel normalization techniques for neural nets nowadays. It doesn't affect the models' performance.

**Comment by glhfgg1024** [comment]
Got it! Thanks a lot!
